benchmarks/aggregate.py

```diff
@@ -36,6 +36,38 @@
 _markers = ('D', 'o', 's')
 
 
+class DatapointSelector:
+
+  def compile(row):
+    total_times = row['metrics']['total_time'] if 'total_time' in row[
+        'metrics'] else []
+    if len(total_times) <= 1:
+      return None
+    compile_and_run_time = total_times[0]
+    run_time = np.average(total_times[1:])
+    std = np.std(total_times[1:])
+    compile_time = compile_and_run_time - run_time
+    # Single sample size for compilation time, std = 0.
+    # Using stddev of the run time distribution.
+    return Datapoint(compile_time, 0)
```

Comment:
@ecota,

I eventually put 0 in here, but in general I'm not sure what to put here, as n = 1 for compilation estimate.

Reply:
0 is the only reasonable value here. stddev of the sample divides by `N-1`, so if N==1 we'd have infinite stddev. Which would be fair enough (we have no clue about the distribution with just one sample), but we're not going to plot infinite error bars :-)

Reply:
Thanks, my reasoning was that if 100% of the data samples (n=1) have the same value then std dev has to be 0, although I felt there some holes in it.